openvino_xai/metrics/insertion_deletion_auc.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any, List, Tuple
+
+import numpy as np
+
+from openvino_xai.explainer.explanation import Explanation, Layout
+from openvino_xai.metrics.base import BaseMetric
+
+
+def auc(arr):
+    """Returns normalized Area Under Curve of the array."""
+    return np.abs((arr.sum() - arr[0] / 2 - arr[-1] / 2) / (arr.shape[0] - 1))
```

Comment:
```suggestion
def AUC(arr):
    """Returns normalized Area Under Curve of the array."""
    return np.abs((arr.sum() - arr[0] / 2 - arr[-1] / 2) / (arr.shape[0] - 1))
```
Could you add the type annotation and doc string?